R/loadFromHumann.R

```diff
@@ -0,0 +1,189 @@
+#' Import HUMAnN results to \code{TreeSummarizedExperiment}
+#
+#' @param file a single \code{character} value defining the file
+#' path of the HUMAnN file. The file must be in merged HUMAnN format.
+#'
+#' @param colData a DataFrame-like object that includes sample names in
+#' rownames, or a single \code{character} value defining the file
+#' path of the sample metadata file. The file must be in \code{tsv} format
+#' (default: \code{colData = NULL}).
+#' 
+#' @param ... additional arguments:
+#' \itemize{
+#'   \item{\code{assay.type}:} {A single character value for naming 
+#'   \code{\link[TreeSummarizedExperiment:TreeSummarizedExperiment-class]{assay}} 
+#'   (default: \code{assay.type = "counts"})}
+#'   \item{\code{removeTaxaPrefixes}:} {\code{TRUE} or \code{FALSE}: Should
+#'     taxonomic prefixes be removed? (default:
+#'     \code{removeTaxaPrefixes = FALSE})}
+#' }
+#'
+#' @details
+#' Import HUMAnN (version 3.0) results of pathways or gene families. Input must
+#' be in merged HUMAnN format. (See the HUMAnN documentation and
+#' \code{humann_join_tables} method.)
+#' 
+#' The function parses gene/pathway information along with taxonomy information
+#' from the input file. This information is stored to \code{rowData}. Abundances
+#' are stored to \code{assays}.
+#' 
+#' Usually the workflow includes also taxonomy data from Metaphlan. See
+#' \link[=loadFromMetaphlan]{loadFromMetaphlan} to load the data to \code{TreeSE}.
+#'
+#' @return  A
+#' \code{\link[TreeSummarizedExperiment:TreeSummarizedExperiment-class]{TreeSummarizedExperiment}}
+#' object
+#'
+#' @name loadFromHumann
+#' @seealso
+#' \code{\link[=loadFromMetaphlan]{loadFromMetaphlan}}
+#' \code{\link[=makeTreeSEFromPhyloseq]{makeTreeSEFromPhyloseq}}
+#' \code{\link[=makeTreeSEFromBiom]{makeTreeSEFromBiom}}
+#' \code{\link[=makeTreeSEFromDADA2]{makeTreeSEFromDADA2}}
+#' \code{\link[=loadFromQIIME2]{loadFromQIIME2}}
+#' \code{\link[=loadFromMothur]{loadFromMothur}}
+#'
+#' @export
+#' @author Leo Lahti and Tuomas Borman. Contact: \url{microbiome.github.io}
+#' 
+#' @references
+#' Beghini F, McIver LJ, Blanco-Míguez A, Dubois L, Asnicar F, Maharjan S,
+#' Mailyan A, Manghi P, Scholz M, Thomas AM, Valles-Colomer M, Weingart G,
+#' Zhang Y, Zolfo M, Huttenhower C, Franzosa EA, & Segata N (2021)
+#' Integrating taxonomic, functional, and strain-level profiling of diverse
+#' microbial communities with bioBakery 3 eLife 10:e65088.
```

Comment:
period after the article title. 

usually journal name with italics (`\emph{... }`?)